admin-auth/src/main/kotlin/com/epam/drill/admin/auth/config/ApiKeyConfig.kt

```diff
@@ -27,4 +27,19 @@ class ApiKeyConfig(private val config: ApplicationConfig) {
      */
     val secretLength: Int
         get() = config.propertyOrNull("secretLength")?.getString()?.toInt() ?: 32
+
+    /**
+     * The maximum number of items or data size that the cache can hold before it starts replacing or removing existing entries.
+     * By default, 1000.
+     */
+    val maximumCacheSize: Long
+        get() = config.propertyOrNull("maximumCacheSize")?.getString()?.toLong() ?: 1000
+
+    /**
+     * The duration in hours after which the stored data in the cache becomes invalid or expired.
+     * By default, 1 hour.
+     */
+    val ttlCacheInHours: Long
+        get() = config.propertyOrNull("ttlCacheInHours")?.getString()?.toLong() ?: 1
```

Comment:
It seems to me that it is better to measure the cache lifetime in minutes rather than in hours, it will be more granular. JWT token expiration is in minutes too.

Reply:
changed